apps/marketing/src/app/components/FeaturesSection/components/FeatureDemo/components/DitheredBackground/DitheredBackground.tsx

```
@@ -0,0 +1,110 @@
+		ctx.putImageData(img, 0, 0);
+	}, [colors]);
+
+	return (
+		<canvas
+			ref={ref}
+			width={W}
+			height={H}
+			className={className}
+			style={{ imageRendering: "pixelated" }}
+			aria-hidden="true"
+		/>
```

Biome (2.4.11)

[error] Disallow aria-hidden="true" from being set on focusable elements.

(lint/a11y/noAriaHiddenOnFocusable)